lib/editor/util/index.js

```diff
@@ -147,6 +152,22 @@ export function getEditorTable (component: string) {
   )
 }
 
+/**
+ * Returns a dictionary from the table spec with the options for route status
+ */
+export function getRouteStatusDict (): any {
```

Comment:
```suggestion
export function getRouteStatusDict (): { [string]: string } {
```
Does this work as a better type than `any`?

Reply:
Flow seems happy with it! :)